internal/services/oracle/oracle_autonomous_database_clone_resource.go

```diff
@@ -0,0 +1,683 @@
+	ResourceGroupName string            `tfschema:"resource_group_name"`
+	Tags              map[string]string `tfschema:"tags"`
+
+	// Required for Clone
+
+	Source       string `tfschema:"source"`
+	SourceId     string `tfschema:"source_id"`
+	CloneType    string `tfschema:"clone_type"`
+	DataBaseType string `tfschema:"data_base_type"`
+
+	// Required (inherited from base)
+
+	AdminPassword                string  `tfschema:"admin_password"`
+	BackupRetentionPeriodInDays  int64   `tfschema:"backup_retention_period_in_days"`
+	CharacterSet                 string  `tfschema:"character_set"`
+	ComputeCount                 float64 `tfschema:"compute_count"`
+	ComputeModel                 string  `tfschema:"compute_model"`
+	DataStorageSizeInTbs         int64   `tfschema:"data_storage_size_in_tbs"`
+	DbVersion                    string  `tfschema:"db_version"`
+	DbWorkload                   string  `tfschema:"db_workload"`
+	DisplayName                  string  `tfschema:"display_name"`
+	LicenseModel                 string  `tfschema:"license_model"`
+	AutoScalingEnabled           bool    `tfschema:"auto_scaling_enabled"`
+	AutoScalingForStorageEnabled bool    `tfschema:"auto_scaling_for_storage_enabled"`
+	MtlsConnectionRequired       bool    `tfschema:"mtls_connection_required"`
+	NationalCharacterSet         string  `tfschema:"national_character_set"`
+	SubnetId                     string  `tfschema:"subnet_id"`
+	VnetId                       string  `tfschema:"virtual_network_id"`
+
+	// Optional for Clone
+
+	CustomerContacts        []string `tfschema:"customer_contacts"`
+	RefreshableModel        string   `tfschema:"refreshable_model"`
+	TimeUntilReconnectClone string   `tfschema:"time_until_reconnect_clone"`
+
+	// optional for clone from backup timestamp
+
+	Timestamp                         string `tfschema:"timestamp"`
+	UseLatestAvailableBackupTimeStamp bool   `tfschema:"use_latest_available_backup_time_stamp"`
+}
+
+func (AutonomousDatabaseCloneResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"location": commonschema.Location(),
+
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ValidateFunc: validate.AutonomousDatabaseName,
+			ForceNew:     true,
+		},
+
+		"resource_group_name": commonschema.ResourceGroupName(),
+
+		// Clone-specific required fields
+
+		"source": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				string(autonomousdatabases.SourceTypeBackupFromTimestamp),
+				string(autonomousdatabases.SourceTypeBackupFromId),
+				string(autonomousdatabases.SourceTypeDatabase),
+				string(autonomousdatabases.SourceTypeCloneToRefreshable),
+			}, false),
+			DiffSuppressFunc: func(k, old, new string, d *pluginsdk.ResourceData) bool {
+				// Source is create-only and not returned by Azure API
+				return old != "" && new == ""
+			},
+		},
+		"source_id": {
```

Comment:
Can we rename this so it's clearer what this ID is supposed to be?

```suggestion
		"source_autonomous_database_id": {
```